plantcv/plantcv/_helpers.py

```diff
@@ -1,11 +1,186 @@
 import cv2
 import numpy as np
+from plantcv.plantcv.dilate import dilate
 from plantcv.plantcv.logical_and import logical_and
+from plantcv.plantcv.image_subtract import image_subtract
 from plantcv.plantcv import fatal_error, warn
 from plantcv.plantcv import params
 import pandas as pd
 
 
+def _find_segment_ends(skel_img, leaf_objects, plotting_img, size):
```

Comment:
@HaleySchuhl it looks like this private function is only used in `segment_ends`, so it can go in that module rather than in `_helpers.py` unless it will be used in other modules eventually?